website/docs/docs/cloud/configure-cloud-cli.md

```diff
@@ -110,13 +110,50 @@ As a tip, most command-line tools have a `--help` flag to show available command
 - `dbt run --help`: Lists the flags available for the `run` command
 :::
  
-### SQLFluff integration
-The dbt Cloud CLI supports [SQLFluff](https://sqlfluff.com/), a modular and configuration SQL linter, which warns you of complex functions, syntax, formatting, and compilation errors.
+### Lint SQL files 
 
-To get started, run `dbt sqlfluff -h` to see the list of supported commands and flags, such as `dbt sqlfluff lint` to lint SQL files.
+From the dbt Cloud CLI, you can invoke [SQLFluff](https://sqlfluff.com/) which is a modular and configurable SQL linter that warns you of complex functions, syntax, formatting, and compilation errors. The available SQLFluff commands are: `fix`, `format`, and `lint`. To show detailed information about these commands and the available flags, run the `dbt sqlfluff -h` command. 
+
+The `lint` command in dbt supports many of the same flags that you can pass to SQLFluff, except for those that might pose a security risk. To lint SQL files, run the command as follows:  
+
+```shell
+dbt sqlfluff lint [PATHS]... [flags]
+```
+
+When no path is set, dbt lints all SQL files in the project. To lint a specific SQL file or a directory of SQL files, set `PATHS` to the path of the SQL file or the directory path, respectively. 
```

Comment:
"When no path is set, dbt lints all SQL files in the current project. To lint a specific SQL file or a directory, set `PATHS` to the path of the SQL file(s) or directory of files you'd like to lint. You can pass multiple `PATHS` to lint multiple files or directories."